PyPoE/cli/exporter/wiki/parsers/item.py

```diff
@@ -3798,6 +3800,23 @@ def _export(self, parsed_args, items):
 
         return r
 
+    def _print_item_rowid(self, parsed_args, base_item_type):
+        #Don't print anything if not running in the rowid mode.
+        if (not 'start' in vars(parsed_args).keys()) or (not 'end' in vars(parsed_args).keys()):
+            return
+
+        export_row_count = parsed_args.end - parsed_args.start
+        #If we're printing less than 100 rows, print every rowid
+        if export_row_count <= 100:
+            print_granularity = 1
+        else:
+            print_granularity = export_row_count//100
+        
+        item_offset = base_item_type.rowid - parsed_args.start
+        if (item_offset == 0) or item_offset % print_granularity == 0:
+            console('Processing item with rowid {}: {}'.format(base_item_type.rowid, base_item_type['Name']))
```

Comment:
Let's try to use the more conventional (nowadays) form of string formatting using f-strings like so:

`console(f'Processing item with rowid {base_item_type.rowid}: {base_item_type['Name']}')`

Keep in mind that f-strings outperform `.format` in a lot of instances. :)

Reply:
Didn't even know these were a thing, let alone the current convention.
Fixed